openapi_python_client/parser/properties.py

```diff
@@ -51,9 +51,16 @@ def get_type_string(self, no_optional: bool = False) -> str:
         Args:
             no_optional: Do not include Optional even if the value is optional (needed for isinstance checks)
         """
-        if no_optional or (self.required and not self.nullable):
-            return self._type_string
-        return f"Optional[{self._type_string}]"
+        return type_string(no_optional, self, self._type_string)
```

Comment:
Could this be a method on `Property` instead of a function that takes in `self`?